test/javascripts/components/gamification-leaderboard-row-test.js

```diff
@@ -0,0 +1,34 @@
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import { query } from "discourse/tests/helpers/qunit-helpers";
+import hbs from "htmlbars-inline-precompile";
+import { module, test } from "qunit";
+import { render } from "@ember/test-helpers";
+
+function displayName() {
+  return query(".user__name").innerText.trim();
+}
+
+module(
+  "Discourse Gamification | Component | gamification-leaderboard-row",
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    test("name", async function (assert) {
```

Comment:
Nitpick, but let's use a more clear description like `Display name prioritizes name` and `Display name prioritizes username`

Reply:
Yessir